lib/v2/screens/explore_screens/explore/interactor/mappers/explore_state_mapper.dart

```diff
@@ -16,18 +16,18 @@ class ExploreStateMapper extends StateMapper {
       var values = results.map((Result element) => element.asValue!.value).toList();
 
       BalanceModel? balanceModel = values.firstWhere((element) => element is BalanceModel, orElse: () => null);
+      PlantedModel? plantedSeeds = values.firstWhere((element) => element is PlantedModel, orElse: () => null);
       VoiceModelAlliance? allianceVoice =
           values.firstWhere((element) => element is VoiceModelAlliance, orElse: () => null);
       VoiceModelCampaign? campaignVoice =
           values.firstWhere((element) => element is VoiceModelCampaign, orElse: () => null);
-      PlantedModel? plantedSeeds = values.firstWhere((element) => element is PlantedModel, orElse: () => null);
 
       return currentState.copyWith(
         pageState: PageState.success,
-        availableSeeds: balanceModel?.roundedQuantity,
+        availableSeeds: balanceModel,
+        plantedSeeds: plantedSeeds,
         allianceVoice: allianceVoice?.amount.toString(),
         campaignVoice: campaignVoice?.amount.toString(),
-        plantedSeeds: plantedSeeds?.quantity.toString(),
       );
```

Comment:
@gguijarro-c-chwy Why is available seeds shown with a rounded value and planted is shown as the quantity is?

Reply:
they should all be rounded